components/fxa-client/src/auth.rs

```diff
@@ -172,3 +198,18 @@ pub struct AuthorizationInfo {
 pub struct MetricsParams {
     pub parameters: HashMap<String, String>,
 }
+
+/// High-level view of the authorization state
+///
+/// This is named `FxaRustAuthState` because it doesn't track all the states we want yet and needs
```

Comment:
Hmmm maybe `InternalAuthState` or something of the sort? I'm also happy to keep it as-is since it seems its temporary

Reply:
Yeah, I think that name would make more sense if it was permanent.  I almost want a name that feels a bit off, to match the temporary situation we're going to be in for the next few months.